packages/mobile/src/screens/app-drawer-screen/useAppDrawerNavigation.ts

```diff
@@ -20,27 +17,35 @@ export const useNotificationNavigation = () => {
     FeatureFlags.MOBILE_NAV_OVERHAUL
   )
 
-  const navigationOptions = isNavOverhaulEnabled
-    ? undefined
-    : {
-        customNativeNavigation: drawerHelpers
-      }
+  const navigation = useNavigation()
 
-  const navigation = useNavigation<
-    AppTabScreenParamList & ProfileTabScreenParamList
-  >(navigationOptions)
+  const notificationNavigation = useMemo(
+    () => ({
+      ...navigation,
+      ...drawerHelpers
```

Comment:
@dylanjeffers this is touching your stuff. It seems that `drawerHelpers` really isn't a navigator and we were only able to treat it as one because of the merging going on in `useNavigation`. I pulled the merge out here so that the types of `useNavigation` could be more accurate

Reply:
sounds good to me, thanks for headsup